### PR TITLE
bug(client): fix mypy exclude typo and some code lint error 

### PR DESCRIPTION
**Pull request by tianweidut**
## Description
- fix mypy exclude typo: ignore some unexpected files
- fix code lint error

## Modules
- [x] Client

## Checklist
- [x] run code format and lint check
- [ ] add unit test
- [ ] add necessary doc

### CODE REVIEW

File: client/starwhale/utils/venv.py [hunk 1]
```diff
@@ -107,12 +108,18 @@ def _do_pip_install_req(
     _hosts: t.List[str] = [_env.get("SW_PYPI_TRUSTED_HOST", "")]
     _index: str = _env.get("SW_PYPI_INDEX_URL", "")
 
+    list_to_str: t.Callable[[t.Union[str, list]], str] = (
+        lambda x: " ".join(x) if isinstance(x, list) else x
+    )
+
+    _pip_index_url = list_to_str(pip_config.get("index_url", ""))
```
Comment (jialeicui) [inline client/starwhale/utils/venv.py:115]: `pip_config.get("index_url", "")` will always returns string?

File: client/starwhale/utils/venv.py [hunk 2]
```diff
@@ -107,12 +108,18 @@ def _do_pip_install_req(
...
-        _extra_index.append(pip_config.get("index_url", ""))
+        _extra_index.append(_pip_index_url)
     else:
-        _index = pip_config.get("index_url", "")
-    _extra_index.extend(pip_config.get("extra_index_url", []))
-    _hosts.extend(pip_config.get("trusted_host", []))
+        _index = _pip_index_url
+
+    _extra_index.append(list_to_str(pip_config.get("extra_index_url", [])))
+    _hosts.append(list_to_str(pip_config.get("trusted_host", [])))
```
Comment (jialeicui) [inline client/starwhale/utils/venv.py:122]: Do we support specify string var to these two fields?

File: client/starwhale/utils/venv.py (hunk 1 above)
Comment (tianweidut) [reply client/starwhale/utils/venv.py:115]: yes， only for mypy check.

File: client/starwhale/utils/venv.py (hunk 2 above)
Comment (tianweidut) [reply client/starwhale/utils/venv.py:122]: ok

**Review by jialeicui — APPROVED** [review]
LGTM

**Pull request merged**